libs/ssl-config/src/main/java/org/elasticsearch/common/ssl/PemUtils.java

```diff
@@ -109,13 +131,9 @@ public static PrivateKey readPrivateKey(Path keyPath, Supplier<char[]> passwordS
             } else if (OPENSSL_EC_PARAMS_HEADER.equals(line.trim())) {
                 return parseOpenSslEC(removeECHeaders(bReader), passwordSupplier);
             } else {
-                throw new SslConfigException("error parsing Private Key [" + keyPath.toAbsolutePath()
-                    + "], file does not contain a supported key format");
+                throw new SslConfigException("cannot read PEM private key [" + keyPath.toAbsolutePath()
+                    + "] because the file does not contain a supported key format");
             }
-        } catch (FileNotFoundException | NoSuchFileException e) {
-            throw new SslConfigException("private key file [" + keyPath.toAbsolutePath() + "] does not exist", e);
-        } catch (IOException | GeneralSecurityException e) {
-            throw new SslConfigException("private key file [" + keyPath.toAbsolutePath() + "] cannot be parsed", e);
```

Comment:
Dumped this exception handling so that the wrapper method above can do a better, more consistent job of it.